local/dev-image/supervisord.conf

```diff
@@ -28,7 +28,17 @@ stderr_logfile_maxbytes=0
 
 # kms derive server
 [program:derive-server]
-command=/app/kms-derive-server --listen-addr 127.0.0.1:1100 
+command=/app/kms-derive-server --listen-addr 127.0.0.1:1100 --docker-compose /app/docker-compose.yml
+autostart=false
+autorestart=true
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile=/dev/stdout
+stderr_logfile_maxbytes=0
+
+# Contract based kms derive server
+[program:derive-server-contract]
+command=/app/kms-derive-server --listen-addr 127.0.0.1:1101 --contract 
```

Comment:
Does this mean contract KMS keys are not deterministic while normal keys are deterministic?